src/test/resources/json/consignments_data_all.json

```diff
@@ -0,0 +1,54 @@
+{
+  "data": {
+    "consignments": [
+      {
+        "exportDatetime": "2020-01-01T09:00:00Z",
+        "seriesid": "fde450c9-09aa-4ba8-b0df-13f9bac1e587",
+        "createdDatetime": "2020-01-01T09:00:00Z",
+        "transferInitiatedDatetime": "2020-01-01T09:00:00Z",
+        "userid": "4ab14990-ed63-4615-8336-56fbb9960300",
+        "consignmentid": "c31b3d3e-1931-421b-a829-e2ef4cd8930c",
+        "consignmentReference": "TEST-TDR-2021-MTB",
+        "totalFiles": 0,
+        "files": [],
+        "fileChecks": {
+          "antivirusProgress": {
+            "filesProcessed": 0
```

Comment:
Given that this is a test of retrieving all data, is it possible to add a few files with a bit of progress? Then we can check that the deferred resolvers are actually calculating something on this endpoint.